### PR TITLE
Update newest Connection points

### PR DESCRIPTION
**Pull request by mheagin**
need to update newest connection points, after the upgrade.

Pull request description
------------------------

### CODE REVIEW

File: gps/turn/index.html [hunk 1]
```diff
@@ -21,20 +21,21 @@ <h2 class="text-left"><i class="fas fa-fw fa-info-circle"></i> About this Networ
 <p>Eighteen <abbr title="National Geodetic Survey">NGS</abbr> Continually Operating Reference Stations (CORS) were selected along the perimeter and interior of the network and were constrained to the published coordinates to conduct the final adjustment of the network coordinates.</p>
 <p>With an activated subscription, set your GPS unit's VRS connection address to <code>168.179.231.9</code> and port <code>2101</code>. Enter your new device-specific username and password.</p>
 <h5>Connections from Source Table</h5>
-<p>For Trimble units we have three different connections.</p>
+<p>For Trimble units we have these different connections.</p>
```
Comment (steveoh) [inline gps/turn/index.html]: Is three any different than these?

File: gps/turn/index.html [hunk 2]
```diff
@@ -21,20 +21,21 @@ <h2 class="text-left"><i class="fas fa-fw fa-info-circle"></i> About this Networ
...
-  <li>VRS_CMR works with older Trimble units.
-  <li>VRS_CMRp works with Trimble units that are newer but may not have the latest updates to the firmware.
-  <li>VRS_CMRx works with the newest Trimble units.
+  <li>VRSCMRX works with the newest Trimble units, newest firmware. 
+  <li>VRSCMRP works with Trimble units that are newer but may not have the latest updates to the firmware.
+  <li>VRSCMR works with older Trimble units.
 </ul>
-<p>All of these give you the network solution.</p>
-<p>For non-Trimble units we have two different connections.</p>
+<p> - - - - </p>
```
Comment (steveoh) [inline gps/turn/index.html]: this should probably be an `<hr />` tag instead of `<p>- - - </p>`

File: gps/turn/index.html [hunk 3]
```diff
@@ -21,20 +21,21 @@ <h2 class="text-left"><i class="fas fa-fw fa-info-circle"></i> About this Networ
...
+<p>For non-Trimble units us these connections.</p>
```
Comment (steveoh) [inline gps/turn/index.html]: use is spelled wrong

File: gps/turn/index.html (hunk 1 above)
Comment (mheagin) [reply gps/turn/index.html]: It may change later down the road, wanted to eliminate a number.

**Pull request merged**